prow/plugins.yaml

```diff
@@ -316,6 +316,15 @@ require_matching_label:
   prs: true
   regexp: ^(sig|wg|committee)/
   # Define needs-sig separately for issues so that we only comment on issues.
+- missing_label: needs-triage
+  org: kubernetes
+  repo: kubernetes
+  issues: true
+  regexp: ^(lifecycle)/(ready|active|frozen)$
```

Comment:
Sorry, I may be someone who helped suggest the use of lifecycle and now I'm not so sure given the interaction between this choice of lifecycles and [the periodic stale->rotten->close jobs](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/test-infra/fejta-bot-periodics.yaml#L163)

The implication here is that issues are going to fall back into `needs-triage` after 90 days (via `lifecycle/stale`) unless they are marked as `lifecycle/frozen`.  That feels like it will lead to increased noise and more things ending up frozen.

Reply:
(replied below)